Documentation/api-reference.md

```diff
@@ -0,0 +1,47 @@
+# Adobe Experience Platform Edge Bridge for Android
+
+## API Reference
+
+| APIs                                           		|
+| ---------------------------------------------- |
+| [extensionVersion](#extensionversion)	|
+| [registerExtension](#registerextension)	|
+
+------
+
+### extensionVersion
+Returns the version of the Edge Bridge extension.
+
+#### Java
```

Comment:
Any chance to add Kotlin examples as well (here and for registration)?

Reply:
Added Kotlin examples to API reference for both methods